litentry/core/stf-task/receiver/src/stf_task_receiver.rs

```diff
@@ -85,13 +85,26 @@ where
 				)
 				.map_err(|e| Error::OtherError(format!("error verify web3: {:?}", e)))?;
 
-				let c = context.create_verify_identity_trusted_call(
+				let c = context.create_identity_verify_trusted_call(
 					request.who,
 					request.identity,
 					request.bn,
 				)?;
 				let _ = context.submit_trusted_call(&c)?;
 			},
+			RequestType::RulesetVerification(request) =>
+				for identity in request.vec_identity {
+					let result = lc_ruleset_build::ruleset_verify(
```

Comment:
I think we have to either use the name `ruleset_verification` or `verify_ruleset` to be more consistent 😂 

Anyways I'm not sure if we are "verifying" ruleset here: we generate a VC with respect to a given ruleset ID, e.g. we can be asked to provide the total amount of DOTs an account has, in that case, we don't verify anything but rather do computation and provide the result. - maybe "build" or "compute" is a better name to avoid confusion, but I don't know

Reply:
Yes. I think it makes more sense with "build".

Reply:
Do you think we should change the name `ruleset` to `assertion`, in order to avoid any confusion?

Reply:
> Do you think we should change the name `ruleset` to `assertion`, in order to avoid any confusion?

This topic was discussed before. Either way is fine for me. But looks like everybody is still talking about `ruleset`. So can we make an alignment again within/across the team?

Reply:
> Do you think we should change the name `ruleset` to `assertion`, in order to avoid any confusion?

I'm also fine with either.

Btw if we replace `ruleset` with `assertion`, what's the name of the single statement inside `assertion`?

Reply:
Have to tag @jingleizhang 😂

Reply:
What about assertion_set and assertion?